tests/requirements.txt

```diff
@@ -1,6 +1,219 @@
-pytest==7.2.2
-pytest-lazy-fixture==0.6.3
-python-dotenv==1.0.1
-jsonschema==4.17.3
-requests==2.32.4
-prometheus_client==0.22.0
+#
+# This file is autogenerated by pip-compile with Python 3.10
+# by the following command:
+#
+#    pip-compile --generate-hashes --output-file=tests/requirements.txt tests/requirements.in
```

Comment:
Please add this to the documentation and include a section on how to install pip-tools (either a reference to environment-dev.yml or an explicit reference to this package)

Reply:
For sure, will do. Dependabot will handle this step entirely, but we should at least know how to run it ourselves.